src/vm/threadsuspend.cpp

```diff
@@ -4987,10 +4984,19 @@ HRESULT ThreadSuspend::SuspendRuntime(ThreadSuspend::SUSPEND_REASON reason)
             {
                 FastInterlockOr((ULONG *) &thread->m_State, Thread::TS_GCSuspendPending);
                 countThreads++;
+
+#if defined(FEATURE_HIJACK) && defined(PLATFORM_UNIX)
```

Comment:
Looks like the comment above (about using GCPOLL) should be updated since now we have a scenario where thread suspension is disabled but we do something other than just rely on polling.